ts/store/reducers/persistedPreferences.ts

```diff
@@ -49,7 +47,6 @@ export const initialPreferencesState: PersistedPreferencesState = {
   preferredLanguage: undefined,
   wasServiceAlertDisplayedOnce: false,
   isPagoPATestEnabled: false,
-  isExperimentalFeaturesEnabled: false,
```

Comment:
If we are removing something from a persisted stare, should not we also need a migration to remove it from the re-hydrated state?

Reply:
Look at migration 18 as an example
```
"18": (state: PersistedState) => {
    const content: ContentState = (state as PersistedGlobalState).content;
    return {
      ...state,
      content: {
        ..._.omit(content, "servicesMetadata")
      }
    };
  },
```

Reply:
Aren't missing keys just ignored?

Reply:
They (should) are not. At rehydration time, it should be loaded from file-storage and set into the in-memory state, even if we are ignoring it. At this point, it should stay in memory (and on disk) until a change in the in-memory state happens. At that time, the persisted state gets updated, removing the old value.

But think at what may happen in this scenario (without a migration): 
- today we have v2.34 in production that stores "isExperimentalFeaturesEnabled" as boolean;
- with this PR, we remove the value without a migration (let us say, release v2.35)
- two releases from now (or more, or less, let us say v2.37), we re-add the same property as a complex object (like `isExperimentalFeaturesEnabled { count: number; state: string; }`). 

Users upgrading from 2.34 (or previous) to v2.37 will have a runtime blocking crash due to the stored state loading a boolean value into an object one (actually, the crash would happen when they try to access the loaded object and find a boolean)

Reply:
@Vangaorth As design engineer I don't have a broad knowledge of these technicalities. This flag was really old and part of the bottom bar logic, so I preferred to remove it completely from the app rather than leave the dead code in the app codebase. If you know how to deal with it without causing issues, contributions are really welcome.

Reply:
Migration pushed with commit [4ba2610ea](https://github.com/pagopa/io-app/pull/4657/commits/4ba2610eaa0f9e4429ba941423c4e93f819dec2b). Tested from v2.34 to v2.35. 
Can somebody else check that the migration is working properly? 
@shadowsheep1 can you double check both the reasoning and the migration correctness?

Reply:
I agree with @Vangaorth indeed. Both reasoning and the migration are correct. I didn't test the migration actually.

Reply:
Should we wait for you confirmation on the migration, @mastro993 and @shadowsheep1? Otherwise I am going to resolve this conversation and give my approval.

Reply:
@Vangaorth let's go ahead! 👍